conf/modules.config

```diff
@@ -35,99 +35,188 @@ process {
         publishDir = [
             path: { "${params.outdir}/reference/intervals" },
             enabled: "params.save_reference",
-            pattern: "*bed"
+            pattern: "*bed",
+            mode: 'copy'
         ]
     }
 
     withName: 'GATK4_CREATESEQUENCEDICTIONARY' {
         publishDir = [
             path: { "${params.outdir}/reference/gatk4" },
             enabled: "params.save_reference",
-            pattern: "*dict"
+            pattern: "*dict",
+            mode: 'copy'
         ]
     }
 
     withName: 'MSISENSORPRO_SCAN' {
         publishDir = [
             path: { "${params.outdir}/reference/msi" },
             enabled: "params.save_reference",
-            pattern: "*list"
+            pattern: "*list",
+            mode: 'copy'
         ]
     }
 
     withName: 'SAMTOOLS_FAIDX' {
         publishDir = [
             path: { "${params.outdir}/reference/fai" },
             enabled: "params.save_reference",
-            pattern: "*fai"
+            pattern: "*fai",
+            mode: 'copy'
         ]
     }
 
     withName: 'TABIX_BGZIPTABIX' {
         publishDir = [
             path: { "${params.outdir}/reference/target" },
             enabled: "params.save_reference",
-            pattern: "*bed.gz"
+            pattern: "*bed.gz",
+            mode: 'copy'
         ]
     }
 
     withName: 'TABIX_DBSNP' {
         publishDir = [
             path: { "${params.outdir}/reference/dbsnp" },
             enabled: "params.save_reference",
-            pattern: "*vcf.gz.tbi"
+            pattern: "*vcf.gz.tbi",
+            mode: 'copy'
         ]
     }
 
     withName: 'TABIX_GERMLINE_RESOURCE' {
         publishDir = [
             path: { "${params.outdir}/reference/germline_resource" },
             enabled: "params.save_reference",
-            pattern: "*vcf.gz.tbi"
+            pattern: "*vcf.gz.tbi",
+            mode: 'copy'
         ]
     }
 
     withName: 'TABIX_KNOWN_INDELS' {
         publishDir = [
             path: { "${params.outdir}/reference/known_indels" },
             enabled: "params.save_reference",
-            pattern: "*vcf.gz.tbi"
+            pattern: "*vcf.gz.tbi",
+            mode: 'copy'
         ]
     }
 
     withName: 'TABIX_PON' {
         publishDir = [
             path: { "${params.outdir}/reference/pon" },
             enabled: "params.save_reference",
-            pattern: "*vcf.gz.tbi"
+            pattern: "*vcf.gz.tbi",
+            mode: 'copy'
         ]
     }
 }
 
+// UMI Subworkflow
+process{
+    withName: 'BAM2FASTQ' {
+        ext.args  = '-T RX'
+    }
+
+    withName: 'SAMBLASTER' {
+        ext.prefix       = {"${meta.id}_unsorted_tagged"}
+        ext.args         = '-M --addMateTags'
+    }
+
+    withName: 'CALLUMICONSENSUS' {
+        ext.args         = '-M 1 -S Coordinate'
+        ext.prefix       = {"${meta.id}_umi-consensus"}
+    }
+
+    withName: 'SAMTOOLS_MERGE' {
+        publishDir       = [ enabled: false ]
```

Comment:
Shouldn't that be the default already?

Reply:
right... I will remove it in all processes

Reply:
removed it everywhere